### ISSUE TITLE
shrunked version for TB 78+

### ISSUE THREAD

**Issue by axelrathey** [darktrojan/shrunked#36]
Hi Darktrojan,

is there a shrunked version for TB 78+? As you wrote max TB version is currently TB73. 
Do you plan to write a shrunked version for TB78+?

Thanks Axel

**Comment by frank-github** [comment]
Hi, would be happy too, if this useful addon can be used for TB78+

**Comment by SHC22** [comment]
Thank you!

**Comment by LampyDave** [comment]
Thank you for your work on this - it's a largely thankless task and fixing all the problems when TB changes things with a new version must be hugely tricky. All very much appreciated.

**Comment by nursoda** [comment]
The current version 5.0.0 works like a charm on 78.x on Arch Linux. THANKS A LOT!!

According to an AMO comment, it seems to break 68.x though on Linux Mint at least.

Nice to have: Settings may be put in a way to support dark mode ;)
![grafik](https://user-images.githubusercontent.com/8425483/95786245-7a0b6080-0cd7-11eb-934a-f96fe66a1e06.png)

**Comment by axelrathey** [comment]
Hi,
many thanks for new version 5.0.0 of Shrunked Image Resizer. In TB 78.3.2 on Windows 10 the addin updated without problems and ist activ now. I will try to send shrunked images with it in the next days.

Axel

**Comment by therob84** [comment]
For me v5.0.0 does not work well with TB 78.4.0 (32 Bit, portable). I made some test, here are my results with the hope that you can solve for next version:

What works:
- setting „resize when attached“ –> attach single or multiple image via TB attachment dialog or drag and drop –> „do you want to resize this/these image(s)?“ –> yes –> OK

What doesn’t work:
- setting „resize when sending e-mail“ –> attach single or multiple image via TB attachment dialog or drag and drop –> sending e-mail –> shrunk-dialogue opens, but only shows one entry for the last image (screenshot). –> clicking OK indeed only resize this last image (1MB-→312KB), but not the other image. Furthermore the e-mail is also NOT sent (what is actually OK), BUT: TBs send button is greyed out now. Only after the e-mail is (auto-)saved as draft (or repoend from draft folder) the send button is back to normal, also the attached images are behaving right only then again by righ-clicking them.
But no way to resize all the other images or send an e-mail at all before not change the settings back to „resize when attach“ (and then manually resize it or re-attach).

system:
- TB: 78.4.0 (32 Bit, portable) [but coming from a years old installation/data folder via migration recently from TB 60→68→78 via auto-update)
- Shrunked Image Resizer 5.0.0

----

I have some more questions:
[BUG?] Settings are not properly saved: minimal file size for resize: 100 kb –> changing to 200 kb is not saved after reopen settings

[QUESTION]: What exactly does „Resize with Javascript“ and what does it mean?
[FEATURE REQUEST]: Could JPEG-quality not set exactly with the common 0...100 slider and number shown?

But in general: thanks for this addon, I formely trusted "Resize Image" add-on on TB60 which unfortunately is not continued thanks to mozillas non understandable API decissions... 

----

**Screenshot 1:** Only last image shown: ![grafik](https://user-images.githubusercontent.com/15350793/97417124-99c19c00-1907-11eb-97a0-ef704699bea9.png)
**Screenshot 2:** Only last image processed: ![grafik](https://user-images.githubusercontent.com/15350793/97417143-a0501380-1907-11eb-9d8b-3ba9364a002f.png)
**Screenshot 3:** Sending button greyed out until draft saved: ![grafik](https://user-images.githubusercontent.com/15350793/97417200-ae9e2f80-1907-11eb-9615-0259c83dd352.png)

**Comment by jpcasta** [comment]
Hello, I am getting the same behavior than the one  therob84 mentioned above.
Regards

**Comment by nursoda** [comment]
For me, on TB 78.4.0 (64-bit), SIR 5.0.0 does NOT yield a dialog when set to "resize when attached", the image was sent in full res (2,7MB). Minimum file size to reduce is set to 200kB, default resize res to 800px. Resample using JS is ON (in case that matters).

**Comment by jiraskosecurenetcz** [comment]
I have same problems with TB 78.4.0. Works fine for one attachment, but multiple images make a issue.

**Comment by darktrojan** [comment]
I've just uploaded version 5.0.1 (not yet approved) which fixes the two major issues reported here. For any other problems please create a new issue.

**Comment by jpcasta** [comment]
Great it works!!
Thanks for modifying the plugin.

**Comment by therob84** [comment]
I can also confirm that it is working (5.0.1 with TB 78.4.3)! Great, thanks!
As a follow-up FeatureRequest I would suggest a third point ("both") of the drop-down-list in settings 1) "when sending e-mail" 2) when attached 3) both.
Is it worth to open as a new ticket for this or is it maybe enough here?
Cheers, Robert
